packages/editor/example/example.materialize.css

```diff
@@ -0,0 +1,157 @@
+/*
```

Comment:
For the example, we use material renderers that use material UI for their styling. Because the "IDE" parts are not using react (yet), we cannot use material UI there. To have material design, we need some parts of materialize CSS. However, we cannot apply the whole materialize.css because it causes issues in combination with material ui. Consequently, for now, an excerpt of materialize.css is provided with the demo.

Reply:
ok, so fix with refactoring